library/src/interactor_impl.cxx

```diff
@@ -964,6 +964,33 @@ interactor& interactor_impl::initCommands()
     std::bind(complNames, std::placeholders::_1,
       std::vector<std::string>{ "field", "array", "component" }));
 
+  this->addCommand(
+    "cycle_point_sprites",
```

Comment:
We really need a generic `cycle xxx` command at some point :)

Reply:
We need this first: https://github.com/f3d-app/f3d/issues/2334